From 56dba3d748d753224bd5aa7e938a5ac3f725e81e Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:24:31 -0400
Subject: [redhat] kabi: Adding symbol param_set_charp (kernel/params.c)

Message-id: <9fe629084f1c409c8e6af727a3581dede817b5ba.1410510773.git.poros@redhat.com>
Patchwork-id: 96096
O-Subject: [PATCH RHEL7.1 151/233] kabi: Adding symbol param_set_charp (kernel/params.c)
Bugzilla: 1071098
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1071098
    https://bugzilla.redhat.com/show_bug.cgi?id=1071098

    Description:
    ============
    Not used directly, used by module_param internally for type char.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/param_set_charp
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/param_set_charp
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/param_set_charp

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/param_set_charp b/redhat/kabi/kabi-module/kabi_ppc64/param_set_charp
new file mode 100644
index 0000000..fc49b8e
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/param_set_charp
@@ -0,0 +1,2 @@
+#1-
+0x62fd6207 param_set_charp vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/param_set_charp b/redhat/kabi/kabi-module/kabi_s390x/param_set_charp
new file mode 100644
index 0000000..fc49b8e
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/param_set_charp
@@ -0,0 +1,2 @@
+#1-
+0x62fd6207 param_set_charp vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/param_set_charp b/redhat/kabi/kabi-module/kabi_x86_64/param_set_charp
new file mode 100644
index 0000000..fc49b8e
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/param_set_charp
@@ -0,0 +1,2 @@
+#1-
+0x62fd6207 param_set_charp vmlinux EXPORT_SYMBOL
-- 
1.7.1